src/main/java/com/example/demo/controller/dto/request/ArticleCreateRequest.java

```diff
@@ -1,10 +1,10 @@
 package com.example.demo.controller.dto.request;
 
 public record ArticleCreateRequest(
-    Long authorId,
-    Long boardId,
+    Long author_id,
+    Long board_id,
     String title,
-    String description
+    String content
```

Comment:
변수명을 안바꾸고 구성할 수도 있습니다.
```suggestion
    @JsonProperty("content")
    String description
```

참고자료: https://www.baeldung.com/jackson-deserialize-snake-to-camel-case#use-jsonproperty-annotation

Reply:
확인했습니다